frontend/src/Components/LoginPage/LoginPage.tsx

```diff
@@ -55,6 +55,7 @@ export const LoginPage = connectToWebSocket(({ socket }) => {
       <input
         id="user"
         type="text"
+        autocomplete="nickname"
```

Comment:
This should fix the bug where credit card information would be suggested on autocomplete, cf. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values